tune_sklearn/tune_search.py

```diff
@@ -143,8 +143,8 @@ class TuneSearchCV(TuneBaseSearchCV):
             this parameter is ignored for ``"bohb"``, as it requires
             ``HyperBandForBOHB``.
 
-        n_iter (int): Number of parameter settings that are sampled.
-            n_iter trades off runtime vs quality of the solution.
+        n_trials (int): Number of parameter settings that are sampled.
```

Comment:
This would clear confusion with `max_iters`, but do you think it'd be helpful to specify that this is the equivalent of `n_iter` from scikit-learn? I remember we discussed naming `max_iters` before too so I'm wondering if it's a problem about this parameter's name or `max_iters`'s.

Reply:
Hm, good point. I think the reference to sklearn doesn't make sense since sklearn doesn't support "iterative models / leveraging partial fit". Since we do, it becomes a source of confusion.